manifest/v1alpha/report/example.yaml

```diff
@@ -0,0 +1,44 @@
+apiVersion: n9/v1alpha
+kind: Report
+metadata:
+  name: report
+  displayName: My report
+spec:
+  timeFrame:
+    snapshot:
+      point: past
+      dateTime: 2024-07-01T10:00:00Z
+      rrule: FREQ=WEEKLY
+    timeZone: America/New_York
+  shared: true
+  filters:
+    projects:
+      - name: project_name
+    services:
+      - name: service_name
+        project: project_name
+    slos:
+      - name: slo_name
+        project: project_name
+        service: service_name
+    labels:
+      label_key_1:
+        - label_value_1
+        - label_value_2
+      label_key_2:
+        - label_value_1
+  systemHealthReview:
+      rowGroupBy: project
+      columns:
+      - displayName: Column 1
+        labels:
+          key1:
+          - value1
+          key2:
+          - value1
+          - value2
+      - displayName: Column 2
+        labels:
+          key3:
+          - value1
+
```

Comment:
remember to add auto generated examples to `internal/manifest/v1alpha/examples` and extend `examplgen` program too ;) you will then be able to get rid of this file entirely